app/2.0/start/toolbox/set-up.md

```diff
@@ -11,39 +11,20 @@ templates for building Progressive Web Apps with Polymer.
 Follow the instructions below to install, build, and deploy a project using an
 App Toolbox template in less than 15 minutes.
 
-## Install the Polymer CLI
+## Install Polymer CLI
 
-Polymer CLI is an all-in-one command line tool for Polymer projects. In this tutorial you use Polymer CLI to initialize, serve, and build your project. You can also use it for linting and testing, but this tutorial won't cover those topics.
+Polymer CLI is an all-in-one command line tool for Polymer projects. In this tutorial you use 
+Polymer CLI to initialize, serve, and build your project. You can also use it for linting and 
+testing, but this tutorial won't cover those topics.
 
-1.  Check that you have already installed all of Polymer CLI's dependencies
-    by running the following commands.
+Polymer CLI requires Node.js, npm and Bower. For full installation instructions, see [the 
```

Comment:
Should include `git`. Technically it's a dependency of Bower rather than ours, but it still needs to be installed.